app/views/prison/visits/show.html.erb

```diff
@@ -0,0 +1,34 @@
+<% content_for :header do %>
+  <%= t('.title') %>
+<% end %>
+
+<%= render(partial: 'people_box') %>
+
+<% if @booking_response.unavailable? %>
+  <h2>
+    <%= t('.unavailable', status: @booking_response.processing_state) %>
+  </h2>
```

Comment:
I think it would be nice to break these two branches out into separate partials.